src/requirements/data/majors/astro.ts

```diff
@@ -0,0 +1,93 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSubRequirements } from '../checkers-common';
+
+const astroRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Introductory Physics Sequence',
+    description: `Three semesters of physics including: 
+    PHYS 1112 or PHYS 1116, PHYS 2213 or PHYS 2217, and PHYS 2214 or PHYS 2218.`,
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19789',
+    checker: includesWithSubRequirements(
+      ['PHYS 1112', 'PHYS 1116'],
+      ['PHYS 2213', 'PHYS 2217'],
+      ['PHYS 2214', 'PHYS 2218']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1],
+    slotNames: ['PHYS 1112 or PHYS 1116', 'PHYS 2213 or PHYS 2217', 'PHYS 2214 or PHYS 2218'],
+  },
+  {
+    name: 'Introductory Mathematics Sequence',
+    description: `Three semesters of math including: 
+    MATH 1910, MATH 1120, or MATH 1220, MATH 1920, MATH 2220, or MATH 2240, and MATH 2930, MATH 4710, or ASTRO 3340`,
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19789',
+    checker: includesWithSubRequirements(
+      ['MATH 1910', 'MATH 1120', 'MATH 1220'],
+      ['MATH 1920', 'MATH 2220', 'MATH 2240'],
+      ['MATH 2930', 'MATH 4710', 'ASTRO 3340']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1],
+    slotNames: [
+      'MATH 1910, MATH 1120, or MATH 1220',
+      'MATH 1920, MATH 2220, or MATH 2240',
+      'MATH 2930, MATH 4710, or ASTRO 3340',
+    ],
+  },
+  {
+    name: 'Experimental or Data Analysis Course in Astronomy',
+    description: `One experimental or data analysis course in astronomy chosen from: 
+    ASTRO 4410, ASTRO 3310, or ASTRO 3334`,
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19789',
+    checker: includesWithSubRequirements(['ASTRO 4410', 'ASTRO 3310', 'ASTRO 3334']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['ASTRO 4410, ASTRO 3310, or ASTRO 3334'],
+  },
+  {
+    name: 'Concentration',
+    description:
+      'In addition to these core requirements, each Astronomy Major must complete a Concentration in either Astrophysics or General Astronomy',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19789',
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      Astrophysics: {
+        description:
+          '5 additional courses in Physics, 3 additional courses in Mathematics, and 2 additional courses in Advanced Astrophysics',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['ASTRO 4431', 'ASTRO 4432', 'ASTRO 4433'],
+          ['PHYS 4230', 'AEP 4230'],
+          ['PHYS 3316', 'PHYS 3318', 'PHYS 3327', 'PHYS 4443'],
+          ['MATH 2940', 'MATH 2210', 'MATH 2230'],
+          ['MATH ****']
+        ),
+        perSlotMinCount: [2, 1, 4, 1, 2],
+        slotNames: [
+          'Two semesters of Advanced Astrophysics',
```

Comment:
Change this to just 'Advanced Astrophysics' bc it looks verbose in the requirements bar
![image](https://user-images.githubusercontent.com/55263191/115173889-36cee780-a096-11eb-8df2-39a703a836b5.png)